src/components/ui/buttons/Button.tsx

```diff
@@ -18,8 +16,6 @@ import { Space } from '../layout/Space';
 import { ProgressIndicator } from '../progress/ProgressIndicator';
 import { ButtonTheme } from './ButtonTheme';
 
-addIcons(faCheck);
```

Comment:
This needs to be added to the imports.
Replace `<Icon name={'check'}/>` with `<Icon name={faCheck}/>`?

Reply:
Yes!